<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadarWindow</class>
 <widget class="QMainWindow" name="RadarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>RadarWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="toolBar">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>240</width>
      <height>120</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>41</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>524287</width>
      <height>524287</height>
     </size>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="features">
     <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
    </property>
    <property name="allowedAreas">
     <set>Qt::NoDockWidgetArea</set>
    </property>
    <property name="windowTitle">
     <string>Toolbar</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2">
     <widget class="QSlider" name="scaleSlider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>800</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="ownSectorSwitcher">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Own Sector</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="centerSeclectBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="scaleLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="menuBar">
   <property name="minimumSize">
    <size>
     <width>74</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>74</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="toolbarOpenBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Toolbar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brightOpenBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bright</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <zorder>menuBar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
